src/lib/autoinstall/widgets/storage/used_as.rb

```diff
@@ -54,9 +54,10 @@ def items
             # TRANSLATORS: option for setting the partition as a RAID member
             [:raid, _("RAID member")],
             # TRANSLATORS: option for setting the partition as an LVM physical volume
-            [:lvm_pv, _("LVM physical volume")]
+            [:lvm_pv, _("LVM physical volume")],
+            # TRANSLATORS: option for setting the partition as a bcache backing device
+            [:bcache_backing, _("Bcache backing device")]
```

Comment:
I guess we should also honor The YaST Upcasing in all the labels of this widget. I just noticed now we are not doing it.

Reply:
If you don't mind, I'll polish those details in a future PR.

Thanks for catching it :)